src/components/Checkout/DonationNudge/DonationNudgeIntro.vue

```diff
@@ -9,6 +9,14 @@
 		>
 			{{ props.header }}
 		</div>
+		<div
+			v-if="props.mobileImageUrl"
+			id="borrower-image-mobile-container"
+			class="hide-for-large"
+			:class="{'has-description': props.description}"
```

Comment:
This is here because when the description exists, we need a tiny bit more margin at the bottom